lib/oppen/utils.rb

```diff
@@ -15,5 +15,30 @@ def self.upsize_circular_array(arr, offset)
       arr.fill(nil, size, 2 * size)
       [arr, 0, size]
     end
+
+    def self.tokens_to_wadler(tokens, base_indent = 4)
```

Comment:
Just remove the `Utils` file, in a new commit, and in the toop-level `Oppen` module, add its functions.